web/models/group.ex

```diff
@@ -22,5 +22,7 @@ defmodule Org.Group do
   def changeset(model, params \\ :empty) do
     model
     |> cast(params, @required_fields, @optional_fields)
+    # |> Ecto.Changeset.validate_required(@required_fields ++ ["user_id"])
```

Comment:
Yes. Once I can upgrade to the latest Ecto and Phoenix, I need that line to ensure groups always have a `user_id`.